web/app/view/StateController.js

```diff
@@ -49,6 +49,9 @@ Ext.define('Traccar.view.StateController', {
                 '#AttributeAliases': {
                     add: 'updateAliases',
                     update: 'updateAliases'
+                },
+                '#Devices': {
+                    remove: 'removeDevice'
```

Comment:
I'm just wondering, why it doesn't get de-selected? Do we really need to explicitly do this?